### ISSUE TITLE
Inconsistent docs about CRDs and K8s version compatibility

### ISSUE THREAD

**Issue by JKolios** [cert-manager/website#388]
Currently, the docs for both version 1.1(https://cert-manager.io/docs/installation/kubernetes/) and 1.2(https://cert-manager.io/next-docs/installation/kubernetes/) read: 
```
# Kubernetes 1.16+
$ kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.1.0/cert-manager.yaml

# Kubernetes <1.16
$ kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/releases/download/v1.1.0/cert-manager-legacy.yaml

    Note: If you’re using a Kubernetes version below v1.15 you will need to install the legacy version of the manifests. This version does not have API version conversion and only supports cert-manager.io/v1 API resources.
```

As a user of k8s 1.15, which CRD version should I install? Is this an inconsistency, or am I reading this section wrong?

**Comment by meyskens** [comment]
You have to use the legacy ones, for more info https://cert-manager.io/docs/release-notes/release-notes-1.0/
A PR to fix this is welcome however in v1.2 Kubernetes 1.15 won't be supported any longer

**Comment by JKolios** [comment]
Looks like it's time for an update then. Thank you.
